cli/pcluster/config/mappings.py

```diff
@@ -463,6 +463,20 @@
                 "allowed_values": ALLOWED_VALUES["per_unit_storage_throughput"],
                 "update_policy": UpdatePolicy.UNSUPPORTED
             }),
+            ("daily_automatic_backup_start_time", {
+                "allowed_values": r"NONE|^([01]\d|2[0-3]):?([0-5]\d)$",
+                "update_policy": UpdatePolicy.SUPPORTED
+            }),
+            ("automatic_backup_retention_days", {
+                "type": IntParam,
+                "allowed_values": "^(3[0-5]|[0-2][0-9]|[0-9])$",
+                "update_policy": UpdatePolicy.SUPPORTED
+            }),
+            ("copy_tags_to_backups", {
+                "type": BoolParam,
+                "default": False,
+                "update_policy": UpdatePolicy.SUPPORTED
+            }),
```

Comment:
`fsx_backup_id` is not included. Are you adding that one later?

Reply:
Yes, that will be covered as part of a separate PR wherein support for create file system from backup will be added.